### DISCUSSION TITLE
Material Design theme input style is not consistent between 'outlined' and 'filled' on input components

### DISCUSSION THREAD

**Discussion by paysonjparker** [primefaces/community#2325] [PrimeNG]
I am trying to use the Input Text component with the theme Material Design in PrimeNg version 17. I also want the theme input style to be set to 'filled'. When I make this change the background of the text area inside the component changes to be filled, but the outline remains. This is not the case in PrimeNg version 16. I would like the 'filled' input style for the Material Design theme to match Material Design's default 'filled' input style.

## Material Design Filled in v17
![image](https://github.com/user-attachments/assets/59650d72-14eb-4194-bf27-fcaad4b212df)

## Material Design Filled in v16
![image](https://github.com/user-attachments/assets/251d6fec-34d2-47c7-b196-9e94c728bf7f)

**Comment by reibanaag** [comment]
Is there any update on this? as I need the one from v16?

**Reply by paysonjparker** [reply]
It looks to me like this still exists unfortunately. To work around this issue, I used custom CSS styling to make it look as close as I could to the v16 Material filled theme.
